.github/ISSUE_TEMPLATE/general.md

```diff
@@ -0,0 +1,23 @@
+---
+name: General
+about: For general Tech Portfolio work
+title: ""
+labels: "g: initial"
+assignees: ""
+---
+
+## Background Information
+
+<!-- Description, links, [user stories](https://www.agilealliance.org/glossary/user-stories/), etc. What is the problem we are trying to solve? If the work is described/tracked elsewhere, feel free to simply link there. Use the [INVEST](https://www.agilealliance.org/glossary/invest) method to assess completeness. -->
+
+## Implementation Steps
+
+<!-- This is the "how": the steps to get the acceptance criteria fulfilled. They can be filled in and adjusted after the issue is created/groomed. -->
+
+- [ ]
+
+## Acceptance Criteria
+
+<!-- Definition of Done. ](https://www.agilealliance.org/glossary/definition-of-done) Should be as objective as possible. -->
```

Comment:
Seems like the hyperlink got a little botched.

```suggestion
<!-- [Definition of Done.](https://www.agilealliance.org/glossary/definition-of-done). This should be as objective as possible. -->
```

Reply:
Never noticed that before! Now you'll have to review this one too 😉  https://github.com/18F/tts-tech-portfolio/pull/1516